OpenSim/Common/Exception.h

```diff
@@ -266,6 +266,11 @@ class IOError : public Exception {
     using Exception::Exception;
 };
 
+class NotImplementedYet : public Exception {
+public:
```

Comment:
This is a generally useful `Exception` type and should:
1. be named more accurately, such as: `MethodNotImplemented` or `MethodNotOverriden` or `OverrideNotImplemented`
2. construct with the method name and format the message consistent with existing exceptions in `Exception.h`